config/deploy.production.yml

```diff
@@ -1,23 +1,28 @@
 # Deploy to these servers.
 servers:
-  - 149.248.53.52
+  - 159.203.62.190
```

Comment:
Note: It would be good to have this as an env variable to ease up open source usage